### PR TITLE
Set deploy app to true

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**

### CODE REVIEW

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-2/1/) has Failed!

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-2/2/) has Failed!

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-2/3/) has Failed!

File: Jenkinsfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
-@Library('github.com/stakater/stakater-pipeline-library@v2.16.26') _
+@Library('github.com/stakater/stakater-pipeline-library@v2.16.22') _
```
Comment (rasheedamir) [inline Jenkinsfile]: why downgrade @ahmedwaleedmalik ?

Comment (ahmedwaleedmalik) [reply Jenkinsfile]: I am testing some stuff out

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-2-4`

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-2-5`

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 6](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-2/6/) has Failed!

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-2-7`

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-2-8`

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-2-9`

**Review by kahootali — APPROVED** [review]

**Pull request merged**